app/src/main/java/com/karumi/kataloginlogoutkotlin/LogInLogOutKata.kt

```diff
@@ -3,20 +3,24 @@ package com.karumi.kataloginlogoutkotlin
 import arrow.core.Either
 import arrow.core.left
 import arrow.core.right
+import kotlinx.coroutines.coroutineScope
```

Comment:
Do we want to add coroutines in this layer? or just add `suspend` keyword?

Reply:
**Nevermind** - you remove it

Reply:
:)